docker/default.conf

```diff
@@ -3,52 +3,8 @@ agents {
     enabled = true
     port = 35000
     dispatchers {
```

Comment:
if these are disabled does it make sense to completely remove these blocks?
if you need to enable one it will force you to declare all of these properties (ie, we lose the defaults).